### ISSUE TITLE
Release a new version of stac-check

### ISSUE THREAD

**Issue by jonhealy1** [stac-utils/stac-check#79]
A new version should be released now that the configuration file option has been added. This version is needed for this pr in stactools: https://github.com/stac-utils/stactools/pull/258

**Comment by jonhealy1** [comment]
@gadomski Should this be a major version release?

**Comment by gadomski** [comment]
Looking at https://github.com/stac-utils/stac-check/compare/v1.1.2...main, I _think_ all of the changes have been API additions. The only potentially breaking change is the name of the entrypoint (`stac_check` to `stac-check`), and I'd say that it's up to you whether you consider that the CLI part of the public API (I usually don't, because I want to maintain flexibility with my CLIs without forcing big version bumps in the package).

Tl;dr: IMO v1.2.0 would be an appropriate version number for the next release.

**Comment by jonhealy1** [comment]
Ok great thank you!
